app/controllers/application_controller.rb

```diff
@@ -16,4 +18,10 @@ def configure_permitted_parameters
   def handle_record_not_found
     render(json: { error: 'Nothing found' }, status: :not_found)
   end
+
+  private
+
+  def set_default_locale
+    I18n.locale = api_v1_user_signed_in? ? current_api_v1_user.locale.to_sym : I18n.default_locale
```

Comment:
out of scope of this task but just so you know
You can use aliases for the default devise helpers so the `api_v1_user` becomes just `user`.
Therefore you could use `current_user` `user_signed_in?` and so on

Reply:
Greate, thanks.